.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,34 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/ubuntu
+{
+	"name": "Ubuntu",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/base:jammy",
+	"features": {
+		"ghcr.io/devcontainers/features/docker-in-docker:2": {}
+	},
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	// "features": {},
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	"forwardPorts": [
+		8080
+	],
+
+	"portsAttributes": {
+		"8080": {
+			"label": "airflow-ui",
+			"onAutoForward": "silent"
+		}
+	},
+
+	// Use 'postCreateCommand' to run commands after the container is created.
+	"postCreateCommand": "git config --global --add safe.directory /workspaces/orca-recipes-airflow",
```

Comment:
@tschaffter: I was getting the following error when I tried `git push`. They suggested that I ran this command, which indeed fixed the issue, but I'm hoping to address the root cause. 

I noticed that the folders in the repo are owned by `root` whereas my user is `vscode`. Did you encounter this and if so, how did you address it? 

```
Git: fatal: detected dubious ownership in repository at '/workspaces/orca-recipes-airflow'
```

Reply:
I [create and configure the user `vscode`](https://github.com/Sage-Bionetworks/sage-monorepo/blob/869abca18aff11af9fa5d682b4ea1a9b494dfcf6/tools/devcontainers/sage/.devcontainer/Dockerfile#L74-L78) in the Dockerfile of the dev container.

Reply:
@BrunoGrandePhD Did you address the root cause of this issue?